utils/src/main/scala/it/pagopa/interop/commons/utils/package.scala

```diff
@@ -18,12 +19,17 @@ package object utils {
   val USER_ROLES: String                                   = "user-roles"
   val CORRELATION_ID_HEADER: String                        = "X-Correlation-Id"
   val IP_ADDRESS: String                                   = "X-Forwarded-For"
+  val ACCEPT_LANGUAGE: String                              = "Accept-Language"
+  val CONTENT_LANGUAGE: String                             = "Content-Language"
   val INTEROP_PRODUCT_NAME: String                         = "prod-interop"
   val PURPOSE_ID_CLAIM: String                             = "purposeId"
   val DIGEST_CLAIM: String                                 = "digest"
   val ORGANIZATION_ID_CLAIM: String                        = "organizationId"
   val SELFCARE_ID_CLAIM: String                            = "selfcareId"
 
+  val OTHER_LANGUAGES: Seq[Language] = Seq(Language("en-US"), Language("en"))
```

Comment:
I may miss something here, but from task shouldn't this be
```suggestion
  val OTHER_LANGUAGES: Seq[Language] = Seq(Language("en-EN"))
```
?